grants.yml

```diff
@@ -2741,14 +2741,8 @@
     - assume:project:releng:ci-group:thunderbird-sheriff
   to:
     - roles:
-        # Geoff Lankow (https://bugzilla.mozilla.org/show_bug.cgi?id=1520433)
-        - login-identity:mozilla-auth0/ad|Mozilla-LDAP|geoff
-        # Magnus Melin (https://bugzilla.mozilla.org/show_bug.cgi?id=1605714)
-        - login-identity:mozilla-auth0/ad|Mozilla-LDAP|mkmelin
-        # Patrick Cloke (https://bugzilla.mozilla.org/show_bug.cgi?id=1595942)
-        - login-identity:mozilla-auth0/ad|Mozilla-LDAP|clokep
-        # Ian Neal (https://bugzilla.mozilla.org/show_bug.cgi?id=1595942)
-        - login-identity:mozilla-auth0/ad|Mozilla-LDAP|iann_cvs
+        # Thunderbird sheriffs, Magnus Melin manager - Bug 1936108
+        - mozilla-group:thunderbird_releng
```

Comment:
Am I missing something or should this be `mozilla-group:thunderbird_sheriff`? (title + https://bugzilla.mozilla.org/show_bug.cgi?id=1933561 seem to indicate that)

Reply:
Nope, you're totally right AFAICT! Thanks for catching that.

Reply:
🤦🏼‍♂️